libs/wire-api-federation/src/Wire/API/Federation/API/Galley.hs

```diff
@@ -70,6 +81,30 @@ newtype GetConversationsResponse = GetConversationsResponse
   deriving (Arbitrary) via (GenericUniform GetConversationsResponse)
   deriving (ToJSON, FromJSON) via (CustomEncoded GetConversationsResponse)
 
+-- | A record type describing a new federated conversation
+data CreateConversation = MkCreateConversation
+  { -- | The time when the conversation was created
+    ccTime :: UTCTime,
+    -- | The user that created the conversation
+    ccOrigUserId :: Qualified UserId,
+    -- | The qualified conversation ID
+    ccCnvId :: Qualified ConvId,
+    -- | The conversation type
+    ccCnvType :: ConvType,
+    -- | The user that created the conversation
+    ccCnvCreator :: Qualified UserId,
```

Comment:
What's the difference between `ccOrigUserId` and `ccCnvCreator`?

Reply:
Per our call, I removed one of them as they're the same.